parcels/field.py

```diff
@@ -133,6 +133,8 @@ def __init__(self, name, data, lon=None, lat=None, depth=None, time=None, grid=N
         self.loaded_time_indices = []
         self.creation_log = kwargs.pop('creation_log', '')
 
+        self.datadepthsize = kwargs.pop('datadepthsize', None)
```

Comment:
Is `datadepthsize` the best name for this variable? Should we explain a bit more what this is for?